sys/include/net/gnrc/ipv6/nc.h

```diff
@@ -293,6 +295,30 @@ static inline bool gnrc_ipv6_nc_is_reachable(const gnrc_ipv6_nc_t *entry)
  */
 gnrc_ipv6_nc_t *gnrc_ipv6_nc_still_reachable(const ipv6_addr_t *ipv6_addr);
 
+/**
+ * @brief   Gets link-layer address from neighbor cache entry if neighbor is reachable.
+ *
+ * @pre (l2_addr != NULL) && (l2_addr_len != NULL)
+ *
+ * @param[out] l2_addr      The link layer address of @p entry. Must not be NULL.
+ * @param[out] l2_addr_len  Length of @p l2_addr. Must not be NULL.
+ * @param[in] entry         A neighbor cache entry
+ *
+ * @return  PID to the interface where the neighbor is.
+ * @return  KERNEL_PID_UNDEF, if @p entry == NULL or the neighbor is not reachable.
+ */
+static inline kernel_pid_t gnrc_ipv6_nc_get_l2_addr(uint8_t *l2_addr, uint8_t *l2_addr_len,
```

Comment:
I would vote for not `inlining` this function. I can imagine that `gnrc_ipv6_nc_get_l2_addr()` will be called from different places  (nd, routing protocols, ...)